util/Migrator/DbScripts/2024-06-17_00_FixOrganizationUserDeleteByIds.sql

```diff
@@ -0,0 +1,108 @@
+IF OBJECT_ID('[dbo].[OrganizationUser_DeleteByIds]') IS NOT NULL
+BEGIN
+    DROP PROCEDURE [dbo].[OrganizationUser_DeleteByIds]
+END
+GO
+
```

Comment:
It is cleaner to use 
CREATE OR ALTER PROCEDURE [dbo].[OrganizationUser_DeleteByIds]
instead of looking up objectid

Reply:
Thank you, updated in https://github.com/bitwarden/server/pull/4197/commits/e4acf9ad35fc3afac754a9b18f19521896cf2a90